src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -339,8 +340,8 @@ internal async Task<ServiceBusReceiver> CreateMessageReceiverAsync()
             EntityName = entityPath;
 
             return string.IsNullOrWhiteSpace(SubscriptionName)
-                ? client.CreateReceiver(EntityName)
-                : client.CreateReceiver(EntityName, SubscriptionName);
+                ? client.CreateReceiver(EntityName, new ServiceBusReceiverOptions { PrefetchCount = Options.PrefetchCount })
+                : client.CreateReceiver(EntityName, SubscriptionName, new ServiceBusReceiverOptions { PrefetchCount = Options.PrefetchCount });
```

Comment:
For some reason, the pre-fetch count was not used. Added it here.

Reply:
Thanks for that